source/Tests/view-old.js

```diff
@@ -0,0 +1,454 @@
+/**
```

Comment:
According to the file's name, this file should not be included in this pool request as well as it should not exist in the repo.